manifests/j/JetBrains/IntelliJIDEA/Community/EAP/212.4535.15/JetBrains.IntelliJIDEA.Community.EAP.yaml

```diff
@@ -0,0 +1,22 @@
+PackageIdentifier: "JetBrains.IntelliJIDEA.Community.EAP"
+PackageVersion: 212.4535.15
+PackageLocale: "en-US"
+Publisher: JetBrains s.r.o.
+PackageName: "IntelliJ IDEA Community (EAP)"
+License: Apache License 2.0
+LicenseUrl: "https://www.apache.org/licenses/LICENSE-2.0.txt"
+PublisherUrl: "https://www.jetbrains.com/idea/"
+PrivacyUrl: "https://www.jetbrains.com/legal/docs/privacy/privacy.html"
+Tags: 
+ - "ide"
+ - "java"
+ - "jetbrains"
+ - "intellij"
+ShortDescription: "IntelliJ IDEA: The Capable & Ergonomic Java IDE by JetBrains"
```

Comment:
```suggestion
ShortDescription: IntelliJ IDEA: The Capable & Ergonomic Java IDE by JetBrains. Note, however, that because EAPs are always works in progress, some of their features might not always work as expected.
```